### DISCUSSION TITLE
Endless renew pop up NVIM (paid account)

### DISCUSSION THREAD

**Discussion by includeovalle** [community/community#30388] [Copilot Conversations]
This menu is popping up every now and then.
I'm a nvim user with a year paid account.
How can I fix this issue?

<img width="936" alt="Screen Shot 2022-08-22 at 7 15 29" src="https://user-images.githubusercontent.com/71602132/185919980-654aedb0-5e6b-4e29-9671-2179bdf47c68.png">

**Comment by mawcc** [comment]
I get this in Visual Studio every couple of minutes. Paid account. Tried reinstalling extension, removing settings directory. "Open Copilot" in the menu leads to an empty window.

![image](https://user-images.githubusercontent.com/3613769/185921148-16ccbf39-3c93-427f-97bb-e8eaa93e88cc.png)
